vtex-apps/checkout-splunk
# CHK-331: Upgrade splunk-events lib and prepare for first release

#### What problem is this solving?

Updates the `splunk-events` library version to latest (which has support for TypeScript), and finish some preparations for the first release.

#### How to test it?

[Workspace](https://lucas--checkoutio.myvtex.com/cart).

You can check the network tab for a splunk request, which will be made as soon as the cart page loads. Alternatively, you can use [this splunk query](https://splunk7.vtex.com/en-US/app/vtex_checkout_ui/search?q=search%20index%3Dcheckout_ui%20workflowInstance%3D%22checkout-cart%22&display.page.search.mode=verbose&dispatch.sample_ratio=1&workload_pool=&earliest=-5h%40h&latest=now&sid=1600449042.5546514_B1AFCE35-A4A4-40A6-92E9-145382F27731) to view the results.

#### Screenshots or example usage:

N/A

#### Describe alternatives you've considered, if any.

N/A

#### Related to / Depends on

N/A
